[Bug]: 参考24.0rc官方文档配置参数启动Qwen3-Reranker模型启动报错

### Your current environment

设备：910B2
固件驱动：26.0.0RC1
镜像：vllm-ascend:v0.23.0rc1 

### 🐛 Describe the bug

https://docs.vllm.ai/projects/ascend/en/v0.24.0rc/tutorials/models/Qwen3-Reranker.html#42-source-code-installation
参考官方配置参数启动Qwen3-Reranker模型启动报错，
配置信息：
#!/bin/sh
vllm serve Qwen/Qwen3-VL-Reranker-2B \
    --served-model-name Qwen/Qwen3-Reranker-0.6B \
    --runner pooling \
    --hf_overrides '{"architectures": ["Qwen3VLForSequenceClassification"],"classifier_from_token": ["no", "yes"],"is_original_qwen3_reranker": true}' \
    --port 8000 \
    --max-model-len 1024
报错信息
args.dispatch_function(args) (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/entrypoints/cli/serve.py", line 148, in cmd (APIServer pid=205)     uvloop.run(run_server(args)) (APIServer pid=205)   File "/usr/local/python3.12.13/lib/python3.12/site-packages/uvloop/__init__.py", line 96, in run (APIServer pid=205)     return __asyncio.run( (APIServer pid=205)            ^^^^^^^^^^^^^^ (APIServer pid=205)   File "/usr/local/python3.12.13/lib/python3.12/asyncio/runners.py", line 195, in run (APIServer pid=205)     return runner.run(main) (APIServer pid=205)            ^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/usr/local/python3.12.13/lib/python3.12/asyncio/runners.py", line 118, in run (APIServer pid=205)     return self._loop.run_until_complete(task) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete (APIServer pid=205)   File "/usr/local/python3.12.13/lib/python3.12/site-packages/uvloop/__init__.py", line 48, in wrapper (APIServer pid=205)     return await main (APIServer pid=205)            ^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/entrypoints/openai/api_server.py", line 665, in run_server (APIServer pid=205)     await run_server_worker(listen_address, sock, args, **uvicorn_kwargs) (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/entrypoints/openai/api_server.py", line 679, in run_server_worker (APIServer pid=205)     async with build_async_engine_client( (APIServer pid=205)                ^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/usr/local/python3.12.13/lib/python3.12/contextlib.py", line 210, in __aenter__ (APIServer pid=205)     return await anext(self.gen) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/entrypoints/openai/api_server.py", line 99, in build_async_engine_client (APIServer pid=205)     async with build_async_engine_client_from_engine_args( (APIServer pid=205)                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/usr/local/python3.12.13/lib/python3.12/contextlib.py", line 210, in __aenter__ (APIServer pid=205)     return await anext(self.gen) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/entrypoints/openai/api_server.py", line 135, in build_async_engine_client_from_engine_args (APIServer pid=205)     async_llm = AsyncLLM.from_vllm_config( (APIServer pid=205)                 ^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/v1/engine/async_llm.py", line 217, in from_vllm_config (APIServer pid=205)     return cls( (APIServer pid=205)            ^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/v1/engine/async_llm.py", line 132, in __init__ (APIServer pid=205)     self.renderer = renderer = renderer_from_config(self.vllm_config) (APIServer pid=205)                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/renderers/registry.py", line 87, in renderer_from_config (APIServer pid=205)     return RENDERER_REGISTRY.load_renderer(renderer_mode, config, tokenizer) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/renderers/registry.py", line 69, in load_renderer (APIServer pid=205)     return renderer_cls(config, tokenizer) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/renderers/hf.py", line 870, in __init__ (APIServer pid=205)     super().__init__(config, tokenizer) (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/renderers/base.py", line 114, in __init__ (APIServer pid=205)     self.mm_processor = mm_registry.create_processor( (APIServer pid=205)                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/multimodal/registry.py", line 230, in create_processor (APIServer pid=205)     return factories.build_processor(ctx, cache=cache) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/multimodal/registry.py", line 95, in build_processor (APIServer pid=205)     return self.processor(info, dummy_inputs_builder, cache=cache) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/multimodal/processing/processor.py", line 992, in __init__ (APIServer pid=205)     self.data_parser = self.info.get_data_parser() (APIServer pid=205)                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/model_executor/models/qwen3_vl.py", line 883, in get_data_parser (APIServer pid=205)     self.get_hf_config().vision_config.spatial_merge_size, (APIServer pid=205)     ^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/model_executor/models/qwen3_vl.py", line 866, in get_hf_config (APIServer pid=205)     return self.ctx.get_hf_config(Qwen3VLConfig) (APIServer pid=205)            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ (APIServer pid=205)   File "/vllm-workspace/vllm/vllm/multimodal/processing/context.py", line 141, in get_hf_config (APIServer pid=205)     raise TypeError( (APIServer pid=205) TypeError: Invalid type of HuggingFace config. Expected type: <class 'transformers.models.qwen3_vl.configuration_qwen3_vl.Qwen3VLConfig'>, but found type: <class 'transformers.models.qwen3.configuration_qwen3.Qwen3Config'> (APIServer pid=205) [ERROR] 2026-08-04-10:23:54 (PID:205, Device:-1, RankID:-1) ERR99999 UNKNOWN applicaiton exception
后来参考0.18版本配置将Qwen3VLForSequenceClassification改成Qwen3ForSequenceClassification后启动正常

--- a/docs/source/tutorials/models/Qwen3-Reranker.md
+++ b/docs/source/tutorials/models/Qwen3-Reranker.md
@@ -120,7 +120,7 @@ If you want to deploy multi-node environment, you need to set up environment on 
     vllm serve Qwen/Qwen3-Reranker-0.6B \
         --served-model-name Qwen/Qwen3-Reranker-0.6B \
         --runner pooling \
-        --hf_overrides '{"architectures": ["Qwen3VLForSequenceClassification"],"classifier_from_token": ["no", "yes"],"is_original_qwen3_reranker": true}' \
+        --hf_overrides '{"architectures": ["Qwen3ForSequenceClassification"],"classifier_from_token": ["no", "yes"],"is_original_qwen3_reranker": true}' \
         --port 8000 \
         --max-model-len 1024
     ```
@@ -132,7 +132,7 @@ If you want to deploy multi-node environment, you need to set up environment on 
     vllm serve Qwen/Qwen3-Reranker-0.6B \
         --served-model-name Qwen/Qwen3-Reranker-0.6B \
         --runner pooling \
-        --hf_overrides '{"architectures": ["Qwen3VLForSequenceClassification"],"classifier_from_token": ["no", "yes"],"is_original_qwen3_reranker": true}' \
+        --hf_overrides '{"architectures": ["Qwen3ForSequenceClassification"],"classifier_from_token": ["no", "yes"],"is_original_qwen3_reranker": true}' \
         --compilation-config '{"cudagraph_capture_sizes": [1024,512]}' \
         --additional-config '{"ascend_compilation_config": {"fuse_norm_quant": false}}' \
         --dtype float16 \
@@ -256,7 +256,7 @@ Here are two accuracy evaluation methods.
                                     dtype="float16",
                                     enforce_eager=True,
                                     max_model_len=10240,
-                                    hf_overrides={"architectures": ["Qwen3VLForSequenceClassification"],"classifier_from_token": ["no", "yes"],"is_original_qwen3_reranker": True})
+                                    hf_overrides={"architectures": ["Qwen3ForSequenceClassification"],"classifier_from_token": ["no", "yes"],"is_original_qwen3_reranker": True})
 
         cache = mteb.ResultCache("/home/data/mteb_data")
         tasks = mteb.get_tasks(

--- a/docs/source/tutorials/models/Qwen3-VL-Reranker.md
+++ b/docs/source/tutorials/models/Qwen3-VL-Reranker.md
@@ -117,6 +117,7 @@ If you want to deploy multi-node environment, you need to set up environment on 
 The Qwen3-VL-Reranker model requires a specific chat template for proper formatting. Create a file named `qwen3_vl_reranker.jinja` with the following content:
 
 ```jinja
+{% raw %}
 <|im_start|>system
 Judge whether the Document meets the requirements based on the Query and the Instruct provided. Note that the answer can only be "yes" or "no".<|im_end|>
 <|im_start|>user
@@ -139,7 +140,7 @@ Judge whether the Document meets the requirements based on the Query and the Ins
     | first
 }}<|im_end|>
 <|im_start|>assistant
-
+{% endraw %}
 ```
 
 Save this file to a location of your choice (e.g., `./qwen3_vl_reranker.jinja`).

